### PR TITLE
Add z-index to top-row

### PR DESCRIPTION
**Pull request by michaelrp**
Added `z-index` to `top-row` class in sample project site.css.

Addresses #22895

### CODE REVIEW

File: src/Components/Samples/BlazorServerApp/wwwroot/css/site.css [hunk 1]
```diff
@@ -24,6 +24,7 @@ app {
     height: 3.5rem;
     display: flex;
     align-items: center;
+    z-index: 10;
```
Comment (javiercn) [inline src/Components/Samples/BlazorServerApp/wwwroot/css/site.css:27]: I think we need to change this in the templates? [here](https://github.com/dotnet/aspnetcore/blob/master/src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/wwwroot/css/site.css) and [here](https://github.com/dotnet/aspnetcore/blob/master/src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/wwwroot/css/app.css)?

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/aspnetcore?pullRequest=24005) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/aspnetcore?pullRequest=24005) before we can accept your contribution.<br/><br/>:x: michaelrp [sign now](https://cla.dotnetfoundation.org/dotnet/aspnetcore?pullRequest=24005)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/aspnetcore?pullRequest=24005) it.</sub>

File: src/Components/Samples/BlazorServerApp/wwwroot/css/site.css (hunk 1 above)
Comment (michaelrp) [reply src/Components/Samples/BlazorServerApp/wwwroot/css/site.css:27]: Updated both.

**Review by javiercn — APPROVED** [review]

**Comment by javiercn** [comment]
Merged! Thanks for the contribution @michaelrp!

**Pull request merged**